activerecord/test/cases/migration_test.rb

```diff
@@ -939,7 +939,15 @@ def quietly
 
   def silence_stream(stream)
     old_stream = stream.dup
-    stream.reopen(RbConfig::CONFIG['host_os'] =~ /mswin|mingw/ ? 'NUL:' : '/dev/null')
+    null = case
+           when IO.const_defined?(:NULL)
```

Comment:
1.9.2 hasn't been supported by Rails for a long time. Even for existing 4.2 build, 1.9.3 is minimum.

As for Rails 5.0, it'll have minimum version of either 2.2 or 2.1.

So I think you could just always use IO:NULL and remove the other conditions